x-pack/plugins/security_solution/server/lib/detection_engine/signals/threat_mapping/enrich_signal_threat_matches.ts

```diff
@@ -92,7 +91,7 @@ export const enrichSignalThreatMatches = async (
     if (!isObject(threat)) {
       throw new Error(`Expected threat field to be an object, but found: ${threat}`);
     }
-    const existingIndicatorValue = get(signalHit._source, DEFAULT_INDICATOR_PATH) ?? [];
+    const existingIndicatorValue = get(signalHit._source, 'threat.indicator') ?? [];
```

Comment:
we could use the previously declared `INDICATOR_DESTINATION_PATH` here instead of the hardcoded value, as this is not a test file

Reply:
I'm inclined to leave this hardcoded string for now, as the check above and the object generation below this line both make assumptions about this string in different ways.